src/main/java/core/basesyntax/Color.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public enum Color {
+    red,
```

Comment:
Since the values in the enum type are constant, you should always represent them in UPPERCASE letters